SDL/MiniDoublet.cu

```diff
@@ -1,86 +1,18 @@
-    
-    mdsInGPU.moduleIndices[idx] = lowerModuleIdx;
-    unsigned int anchorHitIndex, outerHitIndex;
-    if(modulesInGPU.moduleType[lowerModuleIdx] == PS and modulesInGPU.moduleLayerType[lowerModuleIdx] == Strip)
-    {
-        mdsInGPU.anchorHitIndices[idx] = upperHitIdx;
-        mdsInGPU.outerHitIndices[idx] = lowerHitIdx;
-
-        anchorHitIndex = upperHitIdx;
-        outerHitIndex = lowerHitIdx;
-    }
-    else
-    {
-        mdsInGPU.anchorHitIndices[idx] = lowerHitIdx;
-        mdsInGPU.outerHitIndices[idx] = upperHitIdx;
-
-        anchorHitIndex = lowerHitIdx;
-        outerHitIndex = upperHitIdx;
-    }
-
-    mdsInGPU.dphichanges[idx] = dPhiChange;
-
-    mdsInGPU.dphis[idx] = dPhi;
-    mdsInGPU.dzs[idx] = dz;
-    mdsInGPU.shiftedXs[idx] = shiftedX;
-    mdsInGPU.shiftedYs[idx] = shiftedY;
-    mdsInGPU.shiftedZs[idx] = shiftedZ;
-
-    mdsInGPU.noShiftedDzs[idx] = noShiftedDz;
-    mdsInGPU.noShiftedDphis[idx] = noShiftedDphi;
-    mdsInGPU.noShiftedDphiChanges[idx] = noShiftedDPhiChange;
-//#ifdef CUT_VALUE_DEBUG
-//    mdsInGPU.dzCuts[idx] = dzCut;
-//    mdsInGPU.drtCuts[idx] = drtCut;
-//    mdsInGPU.miniCuts[idx] = miniCut;
-//#endif
-
-    mdsInGPU.anchorX[idx] = hitsInGPU.xs[anchorHitIndex];
-    mdsInGPU.anchorY[idx] = hitsInGPU.ys[anchorHitIndex];
-    mdsInGPU.anchorZ[idx] = hitsInGPU.zs[anchorHitIndex];
-    mdsInGPU.anchorRt[idx] = hitsInGPU.rts[anchorHitIndex];
-    mdsInGPU.anchorPhi[idx] = hitsInGPU.phis[anchorHitIndex];
-    mdsInGPU.anchorEta[idx] = hitsInGPU.etas[anchorHitIndex];
-    mdsInGPU.anchorHighEdgeX[idx] = hitsInGPU.highEdgeXs[anchorHitIndex];
-    mdsInGPU.anchorHighEdgeY[idx] = hitsInGPU.highEdgeYs[anchorHitIndex];
-    mdsInGPU.anchorLowEdgeX[idx] = hitsInGPU.lowEdgeXs[anchorHitIndex];
-    mdsInGPU.anchorLowEdgeY[idx] = hitsInGPU.lowEdgeYs[anchorHitIndex];
-
-    mdsInGPU.outerX[idx] = hitsInGPU.xs[outerHitIndex];
-    mdsInGPU.outerY[idx] = hitsInGPU.ys[outerHitIndex];
-    mdsInGPU.outerZ[idx] = hitsInGPU.zs[outerHitIndex];
-    mdsInGPU.outerRt[idx] = hitsInGPU.rts[outerHitIndex];
-    mdsInGPU.outerPhi[idx] = hitsInGPU.phis[outerHitIndex];
-    mdsInGPU.outerEta[idx] = hitsInGPU.etas[outerHitIndex];
-    mdsInGPU.outerHighEdgeX[idx] = hitsInGPU.highEdgeXs[outerHitIndex];
-    mdsInGPU.outerHighEdgeY[idx] = hitsInGPU.highEdgeYs[outerHitIndex];
-    mdsInGPU.outerLowEdgeX[idx] = hitsInGPU.lowEdgeXs[outerHitIndex];
-    mdsInGPU.outerLowEdgeY[idx] = hitsInGPU.lowEdgeYs[outerHitIndex];
-}
-
 void SDL::createMDArrayRanges(struct modules& modulesInGPU, struct objectRanges& rangesInGPU, uint16_t& nLowerModules, unsigned int& nTotalMDs, cudaStream_t stream, const unsigned int& maxPixelMDs)
 {
     /*
-        write code here that will deal with importing module parameters to CPU, and get the relevant occupancies for a given module!*/
+        write code here that will deal with importing module parameters to CPU, and get the relevant occupancies for a given module!
```

Comment:
Isn't this done starting at line 35 of this file? If so, I would delete the comment (I think we have deleted similar ones previously).